rust/crd/Cargo.toml

```diff
@@ -8,8 +8,8 @@ repository = "https://github.com/stackabletech/trino-operator"
 version = "0.3.0-nightly"
 
 [dependencies]
-stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "0.9.0" }
-stackable-regorule-crd = { git = "https://github.com/stackabletech/regorule-operator.git", tag = "0.5.0" }
+stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "0.10.0" }
+stackable-regorule-crd = { git = "https://github.com/stackabletech/regorule-operator.git", branch = "main" }
```

Comment:
Probably want to do a release here?

Reply:
yeah its in the TODO. We have problems with the release script. Will stay draft until regorule was released.